.env.test

```diff
@@ -5,9 +5,8 @@ FLASK_DEBUG=1
 FLASK_SKIP_DOTENV=True
 
 # Change the following values for local development.
-trello_key=kghaksbnsjlaksjdlaksjdldkjlmbsanmbd
-trello_token=asdkjalskjdlaskjdaskjdaskljdlaskjdaskjdaksjdlakjdlaskjdlaskjdlaskjd
-trello_boardid=abcdefg
-todo_list_id=abcdefghijklmnopqrstuvwxyz
-doing_list_id=abcdefghijklmnopqrstuvwxyz
-done_list_id =abcdefghijklmnopqrstuvwxyz
\ No newline at end of file
+MONGO_CONN=mongodb+srv://rajrahman:NWyZIpz9kFnOW4dn@corndel.gyf3r.mongodb.net/test?w=majority
+MONGO_DB_NAME=DevOps
+MONGO_LIST_TODO=todo
+MONGO_LIST_INPROGRESS=inprogress
+MONGO_LIST_DONE=done
```

Comment:
It would be good to also update your `.env.template` to keep that file up to date with these variables